scripts/shell/bt_nus_shell.py

```diff
@@ -1,84 +1,88 @@
-#!/usr/bin/env python
-# Copyright (c) 2019 Nordic Semiconductor ASA
+#!/usr/bin/env python3
```

Comment:
There's nothing wrong with this line, but the commit message still says:

> renamed deprecated python2 version to bt_nus_shell_py2.py

Which is not true anymore